Documentation/concepts.rst

```diff
@@ -377,6 +377,9 @@ identities are prefixed with the string ``reserved:``.
 | reserved:host       | The host network namespace on which the pod or    |
 |                     | container is running.                             |
 +---------------------+---------------------------------------------------+
+| reserved:cluster    | Any network endpoint inside of the cluster that   |
```

Comment:
>Any network endpoint inside of the cluster

How do you distinguish between `cluster` and `world`?

Reply:
inside vs. outside cluster.